cypress-io/cypress
# fix: Restart server on blockHost configuration change

- Closes https://github.com/cypress-io/cypress/issues/22634
### User facing changelog
Cypress will now restart on changes to `blockHosts` configuration entry. Fixes #22634

### Additional details
Changes to `blockHosts` configuration entry should trigger a restart and result in new values being properly loaded.

### Steps to test
Launch cypress, modify `blockHosts` entry, wait for cypress to restart and see if new values are used.

### How has the user experience changed?
Can now change `blockHosts` entry without the need to manually rerun cypress.

### PR Tasks
